include/cling/Utils/AST.h

```diff
@@ -50,6 +50,18 @@ namespace utils {
     ///
     bool IsWrapper(const clang::FunctionDecl* ND);
 
+    ///\brief Get the mangled name of a NamedDecl.
+    ///
+    ///\param [in]  ND - Try to mangle this decl's name.
+    ///\param [in]  GD - Provide the GlobalDecl if what is to be mangled is a
+    /// constructor or destructor.
+    ///
+    ///\returns The mangled or pure name (when ND should not be mangled) on
+    /// success or an empty string on failure.
+    ///
+    std::string maybeMangleDeclName(const clang::NamedDecl* ND,
+                                    const clang::GlobalDecl* GD = nullptr);
```

Comment:
Then I am a bit confused (i.e. I am missing something).  d14e9b4 seems to be in the master for a long while, while this PR seems to not include it ... what am I missing?